packages/calcite-components/src/assets/styles/_sortable.scss

```diff
@@ -9,20 +9,17 @@
 
   .calcite-sortable--ghost::before {
     content: "";
-    @apply box-border
-      border
-      border-dashed
-      border-color-brand
-      bg-foreground-2
-      absolute
-      top-0
-      left-0
-      bottom-0
-      right-0
-      z-default;
+    box-sizing: border-box;
+    border-width: var(--calcite-border-width-sm);
+    border-style: dashed;
+    border-color: var(--calcite-color-brand);
+    background-color: var(--calcite-color-foreground-2);
```

Comment:
Should this allow for a passed component token similar to `_spacing`?

Reply:
Unfortunately, we can't use component tokens for sortable.js. The library uses global classes to style elements so none of these sortable styles are scoped to any one single component.

However, users can restyle these classes since they are global classes. We could have global root vars for these if we want to allow them to style that way. what do you think @jcfranco?

For now, i'd probably like to hold off on that though since these styles are going to change with some upcoming list design changes.